src/csharp/Grpc.Examples.MathClient/project.json

```diff
@@ -5,6 +5,10 @@
   "configurations": {
     "Debug": {
       "buildOptions": {
+        "define": [ "SIGNED" ],
```

Comment:
There's the obvious sharing of json values, but I didn't see any way to have global, shared json values.